src/Widgets/PowerModeList.vala

```diff
@@ -0,0 +1,170 @@
+/*
+ * SPDX-License-Identifier: GPL-2.0-or-later
+ * SPDX-FileCopyrightText: 2024 elementary, Inc. (https://elementary.io)
+ */
+
+using Power.Services.DBusInterfaces;
+
+public class Power.Widgets.PowerModeList : Gtk.Box {
+    public static bool successfully_initialized { get; private set; default = true; }
+
+    private static PowerProfile? pprofile;
+    private static GLib.Settings? settings;
+
+    public string settings_key { get; construct; }
+
+    private Gtk.RadioButton saver_radio;
+    private Gtk.RadioButton balanced_radio;
+    private Gtk.RadioButton performance_radio;
+
+    private const string ICON_RES = "/io/elementary/desktop/wingpanel/power/scalable/categories/";
+
+    public PowerModeList (bool on_battery) {
+        Object (
+            settings_key: on_battery ? "profile-on-good-battery" : "profile-plugged-in",
+            orientation: Gtk.Orientation.VERTICAL,
```

Comment:
You can use only the actual name of the constant for a property that takes a namespaced constant like 
```suggestion
            orientation: VERTICAL,
```

Reply:
That's not working 😕

Reply:
Ah maybe it doesn't work in Object calls, sorry. Either way, previous comment about only having construction properties in the object call still stands! If a new constructor is added in the future for example these should really be in the `construct` block

Reply:
Done